website/docs/cairo-1/02-understanding-cairo-packages.md

```diff
@@ -3,7 +3,10 @@
 There are several requirements that Cairo packages have to follow. These are explained in the following sections.
 
 You can refer to [official Cairo documentation](https://github.com/starkware-libs/cairo/tree/main/docs/reference) for
-more details.
+more details. 
+
+Keep in mind that Protostar does not support `cairo_project.toml`. 
+It uses [Scarb](https://docs.swmansion.com/scarb) - a powerful package manager -  and its [manifest files](https://docs.swmansion.com/scarb/docs/reference/manifest) instead.
```

Comment:
```suggestion
It uses [Scarb](https://docs.swmansion.com/scarb) and its [manifest files](https://docs.swmansion.com/scarb/docs/reference/manifest) instead.
```

that part was non-technical and superfluous

Reply:
Yea, those are more (intended) advertisements of Scarb, you are right xD